fortran/nf_lib.c

```diff
@@ -25,203 +25,27 @@
 #ifdef USE_NETCDF4
 
 /**
```

Comment:
What happens to these functions? This looks like a great refactor...

Reply:
They are not needed.  It is all handled on the Fortran side.